server/images.go

```diff
@@ -1162,3 +1164,24 @@ func verifyBlob(digest string) error {
 
 	return nil
 }
+
+var isContainer = ""
+
+func init() {
+	if runtime.GOOS == "linux" {
+		file, err := os.Open("/proc/1/cgroup")
```

Comment:
not sure how official this is but checking `.dockerenv` would be an easy check

Reply:
Based on the issues I see come in, quite a few folks in the community use alternative container runtimes to Docker, so I don't think that approach will work.